### PR TITLE
371 version bump automation

### LINKED ISSUE #372: Add version number bumber, for example grunt-bump

### PR DESCRIPTION
**Pull request by kronken**
Closes #372

### CODE REVIEW

File: .yarnrc [hunk 1]
```diff
@@ -0,0 +1 @@
+ignore-engines true
```
Comment (neodyymi) [inline .yarnrc]: Why is this necessary?

Comment (kronken) [reply .yarnrc]: It is necessary due to a known issue with a dependency that bumped uses. (cson-parser)
https://github.com/groupon/cson-parser/issues/74

Comment (neodyymi) [reply .yarnrc]: Let's add a comment to explain this

Comment (kronken) [reply .yarnrc]: done

**Comment by neodyymi** [comment]
Looks good!
